log.go

```diff
@@ -360,12 +368,14 @@ func (l *Logger) WithLevel(level Level) *Event {
 		return l.Info()
 	case WarnLevel:
 		return l.Warn()
+	case NotifyLevel:
+		return l.Notify()
 	case ErrorLevel:
 		return l.Error()
-	case FatalLevel:
-		return l.newEvent(FatalLevel, nil)
-	case PanicLevel:
-		return l.newEvent(PanicLevel, nil)
+	case CriticalLevel:
+		return l.newEvent(CriticalLevel, nil)
```

Comment:
why not `l.Critical()` and same for alert?

Reply:
The original behaviour of the lib is:
* `l.Critical` would log an event with severity critical and also kill the app (actually this was valid for `Fatal` and `Panic`)
* `l.WithLevel(Critical)` would create an event with severity critical, but it wouldn't kill the app.

That's why they are created with `newEvent` instead of `l.X`. I can change it to use the `l.X` methods